### DISCUSSION TITLE
How's Vite support?

### DISCUSSION THREAD

**Discussion by estaub** [vanilla-extract-css/vanilla-extract#289] [Q&A] [answered]
The Vite issue referenced in [the Vite setup warning](https://vanilla-extract.style/documentation/setup/#vite) is fixed, and I've seen that there have been a few other issues (apparently fixed) with the Vite plugin recently.  I want to kick the tires on vanilla-extract and sprinkles.  Is starting with Vite a good idea at the moment?

**Comment by rtkaaho** [comment]
Currently playing around with vite. Other than the caveats mentioned in the docs here: https://vanilla-extract.style/documentation/setup/#vite it works pretty well. Running in watch mode seems to be causing some issues for me on subsequent rebuilds. I am using vite in `lib` mode though since I am building shared components using vanilla-extract.

**Comment by mattcompiles** [comment, marked as answer]
All the Vite specific issues I'm aware of are now fixed. I'd have no issues recommending giving vanilla-extract a go with Vite. 👍
